<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Widget{
	border-image: url(:/C:/share/OIP-C (1).jpg);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_connect">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>连接mqtt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_usrname">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>desk</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_client">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>客户端名字：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="popTopic">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>aaa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hostaddr">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>主机地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_usrname">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_client">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>widget</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="msgEitd_pub">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_hostaddr">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6666</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>订阅主题：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sub">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>189</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ccc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pub">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>主题格式：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pub">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发布</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hostport">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>主机端口：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_hostport">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1883</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sub">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>订阅</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>30</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>光照强度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_GY">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>609</x>
     <y>30</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>烟雾浓度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_PPM">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>温度:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TEMP">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>湿度:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_HUMI">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>开门</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addcard">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>增加卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delcard">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>减少卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setvalue">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>设置阈值</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_setvalue">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>卡号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_add">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>301</y>
     <width>781</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
